initial commit

alangre3n opened the pull request:
Hi

alangre3n committed:
initial commit

Vanadiiii commented on src/main/java/ru/dexsys/App.java:
```diff
@@ -0,0 +1,14 @@
+{
+    public static void main( String[] args ) {
+        WebDriver driver = new ChromeDriver();
+        WeatherPage page = new WeatherPage(driver);
```
like!)

Vanadiiii commented on src/main/java/ru/dexsys/WeatherPage.java:
```diff
@@ -0,0 +1,38 @@
+    public void getTempTenDays() {
+        List<WebElement> dayNumber = webDriver.findElements(By.xpath("//strong"));
+        List<WebElement> monthAndDayOfWeek = webDriver.findElements(By.xpath("//small"));
+        List<WebElement> tempDayPart = webDriver.findElements(By.className("weather-table__body-cell_type_daypart"));
```
Лучше определить коллекцию элементов, у которых есть //strong, //small и 4 'weather-table......'

Vanadiiii reviewed (commented):
Объедини элементы, и будет совсем здорово

alangre3n committed:
замечание исправлены
теперь все элементы кладутся в одну коллекцию
добавлен метод для удаления лишних элементов
добавлен метод для красивого вывода температуры на экран
